src/azure.rs

```diff
@@ -241,6 +241,14 @@ impl ObjectStore for MicrosoftAzure {
             objects,
         })
     }
+
+    async fn copy(&self, _from: &Path, _to: &Path) -> Result<()> {
+        todo!()
```

Comment:
Perhaps we could return `Err(crate::Error::NotImplemented)` instead of panicking